simulator2/scheme-adapter.env

```diff
@@ -50,9 +50,9 @@ CACHE_PORT=6379
 
 # SWITCH ENDPOINT
 # The option 'PEER_ENDPOINT' has no effect if the remaining three options 'ALS_ENDPOINT', 'QUOTES_ENDPOINT', 'TRANSFERS_ENDPOINT' are specified.
-PEER_ENDPOINT=mojaloop-testing-toolkit:5000
-#PEER_ENDPOINT=172.17.0.1:5000
-#PEER_ENDPOINT=docker.for.mac.localhost:5000
+PEER_ENDPOINT=mojaloop-testing-toolkit:4040
+#PEER_ENDPOINT=172.17.0.1:4040
```

Comment:
Please add reasons for commented-out lines here, or otherwise remove them. Applies to other lines below...